### DISCUSSION TITLE
[Feature] Automatically rotate the photos as they should be.

### DISCUSSION THREAD

**Discussion by PaulEins** [immich-app/immich#14064] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Great idea that many will like.

If you have activated the orientation lock on the iPhone. Then the camera doesn't know how the phone was held while taking the picture.

And, even if the orientation lock is normal, if you take a photo facing downwards, this is not recognized correctly. 

This means that many photos have to be rotated manually. But an AI or a tool can simply recognize how it has to be turned. That would be a function that many people would appreciate. I've never seen anything like it. 

Translated with DeepL.com (free version)

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#8355

**Reply by PaulEins** [reply]
I have read through your reference. I do not recognize my concern. Above it is about photos where the alignment cannot be read correctly.

My concern is that the alignment is recognized on the basis of the image. A car is not upside down or lettering should not go upwards. Shoes are at the bottom. The mouth is not up.

Because of the screen lock, the image is not provided with the information. That's the point. 

Translated with DeepL.com (free version)

**Reply by bo0tzz** [reply]
Then #10136
